packages/utils/src/lib/reports/__snapshots__/report.md

```diff
@@ -188,223 +358,445 @@ ESLint rule **eqeqeq**. [📖 Docs](https://eslint.org/docs/latest/rules/eqeqeq)
 
 ### Disallow `target="_blank"` attribute without `rel="noreferrer"` (ESLint)
 
-🟩 <b>passed</b> (score: 100)
+<details>
+<summary>🟩 <b>passed</b> (score: 100)</summary>
+
+
+
+</details>
+
```

Comment:
:lady_beetle: We don't want to render a `details` element when there isn't any additional info.

Reply:
fixed!!